pkg/register/register.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2017 by the contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package register implements automatic registration of webhooks and branch
+// protections on all DCO repositories in a set of organizations.
+package register
```

Comment:
suggestion: a package's name should be a one word description of what it provides. webhooks might be a better name than register, the verb.